docs/becoming-a-maintainer/translations/es/problemas-y-solicitudes-de-extraccion.md

```diff
@@ -0,0 +1,181 @@
+
+#### Errores de pequeños a medianos
+
+Un error que no afecta la funcionalidad de tu aplicación no se considera crítico. Puedes categorizar los errores como de nivel medio cuando puedes corregirlos después de desplegar un lanzamiento próximo. Aquellos que puedas o no incluir en el desarrollo se consideran errores pequeños.
+
+Puedes abrir estos problemas para que cualquiera trabaje en ellos. Algunos de estos problemas pueden ser tan pequeños que serían una buena oportunidad para un nuevo colaborador. Etiqueta el problema con un `buen primer problema` o `solo para principiantes` en esta situación.
+
+### Problemas etiquetados como características
+
+Al priorizar las solicitudes de características, debes asegurarte de que sean adecuadas para tu proyecto y que te interese agregarlas. Comunícate con el autor original del problema para ver si quiere trabajar en él. Si están de acuerdo, continúa y asígnale ese problema. De lo contrario, agrega una etiqueta `se necesita ayuda` o `aceptando PR`.
+
+Supongamos que la función es adecuada para el proyecto pero es compleja. Considera trabajar en ella tú mismo o hacer que un miembro del equipo central la asigne. Las funciones grandes que afectan muchos archivos y partes móviles de tu aplicación deben ser manejadas por alguien experimentado con la base de código y no crearán más problemas.
+
+A veces, los usuarios pedirán funciones que ya están en la hoja de ruta o que ya están siendo trabajadas por otro colaborador. Si es así, puedes responder cortésmente para informarles sobre el estado. Aquí hay una plantilla que puedes usar para una respuesta:
+
+> Gracias por tomarte el tiempo para abrir este problema. Otro miembro del equipo está trabajando en esta función, que se agregará pronto. Como resultado, vamos a cerrar este problema.
+
+Si una solicitud de característica no suena como una buena opción para tu proyecto, puedes responder al autor original y cerrar el problema. Aquí hay una plantilla que puedes usar para una respuesta:
+
+> Gracias por estar tan interesado en nuestro proyecto. La función que has propuesto no sería una buena opción para el alcance y la dirección actuales de este proyecto. En este momento, no vamos a avanzar con esta función.
+
+### Problemas etiquetados como documentación
+
+Si el problema tiene que ver con errores tipográficos o gramaticales menores, agrega una etiqueta `buen primer problema` para abrirlo a nuevos colaboradores. Si el problema es más complejo, deberás evaluar el nivel de prioridad y agregar una etiqueta `se necesita ayuda`. Además, en estas situaciones, es posible que debas obtener más información del autor original antes de abrirlo para la contribución.
```

Comment:
similar comment about needing to decide if we want to keep all labels in english or translate. It looks like in this file all labels have been translated so maybe thats what we have decided on.